pkg/controller/v1beta1/inferenceservice/controller_test.go

```diff
@@ -722,307 +722,6 @@ var _ = Describe("v1beta1 inference service controller", func() {
 		})
 	})
 
-	Context("Inference Service with explainer", func() {
```

Comment:
Should we add a test with art explainer ?

Reply:
agreed, let's keep the test with art explainer